spring/1.3/modules/ROOT/pages/spring-module-reference.adoc

```diff
@@ -0,0 +1,59 @@
+= Spring Module Reference - Mule 4
+
+Spring module v1.3
+
+The Spring module enables Mule apps to use the Spring framework.
+
+== Configurations
+---
+
+[[config]]
+=== Config
+Spring configuration that enables you to define a set of spring XML files and create an application context with objects to use in the Mule artifact.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Files a| String |  |  | x
+|===
+
+== Operations
+
+[[authorization-filter]]
+=== Authorization Filter
+
+`<spring:authorization-filter>`
+
+Authorize users against a required set of authorities.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Required Authorities a| String |  |  | x
+|===
+
+==== Throws
+
+* MULE:NOT_PERMITTED
+* MULE:SERVER_SECURITY
+
+== Types
+
+[[DelegateSecurityProvider]]
+
+=== Delegate Security Provider
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Name a| String | Provider name to reference. |  | x
+| Delegate Ref a| String | Reference to a Spring Security Manager to use. |  | x
+| Authentication Provider Ref a| String | Reference to an authentication provider to use. |
```

Comment:
You don't need to say "Reference to a". Can you say something like this? "Authentication provider that authenticates..." If that is too much, just say "Authentication provider."